scripts/util/resolve-types.js

```diff
@@ -59,7 +59,7 @@ module.exports = (shapesWithoutNames, { inputShapes, outputShapes }) => {
       decoder: `(${jsonDecode}.list ${child.decoder})`,
       jsonEncoder: `(List.map (${child.jsonEncoder})) >> ${jsonEncode}.list`,
       queryEncoderType: child.queryEncoderType,
-      queryEncoder: base => `AWS.Encode.addListToQueryArgs (${child.queryEncoder(base)}) "${base}"`,
+      queryEncoder: base => `AWS.Encode.addListToQueryArgs (${child.queryEncoder('')}) "${upCam(base)}"`,
```

Comment:
Was getting `AttributeNames.0.AttributeNames=All`, but it should be `AttributeNames.0=All`. Will have to also test when a record is nested